axes/tests/test_access_attempt.py

```diff
@@ -394,3 +394,49 @@ def test_custom_authentication_backend(self):
         request.user = self.user
         authenticate(request=request, foo='bar')
         self.assertEqual(AccessLog.objects.all().count(), 0)
+
+    # by default, AXES_RESET_ON_SUCCESS = False
+    def test_reset_on_success_default(self):
+        """Tests that the failure attempts does not reset after one successful
+        attempt by default.
+        """
+        print('RESET = FALSE')
+        # test until one try before the limit
+        for _ in range(1, settings.AXES_FAILURE_LIMIT):
```

Comment:
Just `range(settings.AXES_FAILURE_LIMIT - 1)` would be cleaner to read IMO, but that's just a preference.

Reply:
I agree. I took it from another test and figured it was the style of this repos. I will rewrite it.